<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasterWidget</class>
 <widget class="QWidget" name="MasterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="serverGroupBox">
     <property name="title">
      <string>Server Configuration</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="portSpinBox">
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>8080</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startServerButton">
        <property name="text">
         <string>Start Server</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopServerButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Stop Server</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="calculationGroupBox">
     <property name="title">
      <string>Calculation Configuration</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="rangeStartLabel">
        <property name="text">
         <string>Range Start:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rangeStartEdit">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rangeEndLabel">
        <property name="text">
         <string>Range End:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rangeEndEdit">
        <property name="text">
         <string>1000000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="distributeButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Distribute Work</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="verifyButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Verify Results</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="clientsGroupBox">
       <property name="title">
        <string>Connected Clients</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="clientsListWidget"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="primesGroupBox">
       <property name="title">
        <string>Found Prime Numbers</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
	    <item>
         <widget class="QLabel" name="primeCountLabel">
          <property name="text">
           <string>Found: 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="primesListWidget"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="logGroupBox">
     <property name="title">
      <string>Log</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTextEdit" name="logTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Server not running</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
